src/config.c

```diff
@@ -3074,6 +3074,7 @@ standardConfig static_configs[] = {
     createBoolConfig("lazyfree-lazy-user-flush", NULL, DEBUG_CONFIG | MODIFIABLE_CONFIG, server.lazyfree_lazy_user_flush, 0, NULL, NULL),
     createBoolConfig("repl-disable-tcp-nodelay", NULL, MODIFIABLE_CONFIG, server.repl_disable_tcp_nodelay, 0, NULL, NULL),
     createBoolConfig("repl-diskless-sync", NULL, DEBUG_CONFIG | MODIFIABLE_CONFIG, server.repl_diskless_sync, 1, NULL, NULL),
+    createBoolConfig("dual-conn-sync-enabled", NULL, DEBUG_CONFIG | MODIFIABLE_CONFIG | HIDDEN_CONFIG | PROTECTED_CONFIG, server.dual_conn_enabled, 0, NULL, NULL),
```

Comment:
Why is this HIDDEN and PROTECTED and also disabled? How are users supposed to enable this?

Reply:
We used the PROTECTED flag to prevent users from accidentally changing the setting in production environments. You're right about the HIDDEN flag, it doesn't add much value here, so we can remove it to make the setting more visible.

The feature is currently disabled by default because we want to gather more feedback before enabling it globally. After we gather sufficient feedback and gain confidence in the feature's stability and usefulness, we plan to remove the PROTECTED flag as well.

Reply:
I think we are also missing the update in the valkey.conf file to describe the feature and how to enable it.